ha-jobs-play/src/main/scala/de/kaufhof/hajobs/OverviewController.scala

```diff
@@ -0,0 +1,21 @@
+package de.kaufhof.hajobs
+
+import play.api.http.ContentTypes
+import play.api.mvc.{Action, Controller}
+import play.api.routing.{JavaScriptReverseRoute, JavaScriptReverseRouter}
+
+class OverviewController(jobsInfoRoute: JavaScriptReverseRoute) extends Controller {
```

Comment:
Do you need the JavaScriptReverseRouter? If so, is only one route needed? Or all?

Maybe some documentation how to include the OverviewController is helpful.